modules/admin/server/routes/admin.server.routes.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var adminPolicy = require('../policies/admin.server.policy'),
+	admin = require('../controllers/admin.server.controller');
+
+module.exports = function(app) {
+	// Users collection routes
+	app.route('/api/users').all(adminPolicy.isAllowed)
```

Comment:
Would it make more sense to create unique routes for the Admin module? For instance, we could define the routes here as '/api/admin/users'

If we define the same routes in different modules, this could get confusing. I understand that aren't any conflicts between the existing user routes, and these new routes. However, I think it might make more sense to use completely separate routes.